tests/cupyx_tests/scipy_tests/special_tests/test_basic.py

```diff
@@ -81,3 +82,36 @@ def test_log1p_complex(self):
         # complex-valued log1p not yet implemented
         with pytest.raises(TypeError):
             cupyx.scipy.special.log1p(0 + 0j)
+
+    @pytest.mark.parametrize("function", ["xlogy", "xlog1py"])
+    @testing.for_float_dtypes()
```

Comment:
How about testing also for complex dtypes?
```suggestion
    @testing.for_dtypes('efdFD')
```

Reply:
yes, this was my intention! (I already had complex values in the data generation below and had tested it locally in a console)